src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -175,6 +160,38 @@ public IReadOnlyCollection<IFile> Install(PackageReference package, PackageType
             return result;
         }
 
+        private static DirectoryPath GetPackagePath(DirectoryPath root, PackageReference package)
+        {
+            if (package.Parameters.ContainsKey("version"))
+            {
+                var version = package.Parameters["version"].First();
+                return root.Combine($"{package.Package}.{version}");
```

Comment:
https://www.nuget.org/packages/ReportGenerator/2.4.5

![image](https://user-images.githubusercontent.com/1647294/28783843-e8f937e8-7611-11e7-8e43-59c9a55293c7.png)


nuget install ReportGenerator -Version 2.4.5

![image](https://user-images.githubusercontent.com/1647294/28783964-49f40ce4-7612-11e7-8152-598580d9ed4d.png)

It doesn't always happen, only when a package has done the x.x.x.x in nuspec
```xml
<version>2.4.5.0</version
```